docs/howto/windows.txt

```diff
@@ -4,27 +4,26 @@ How to install Django on Windows
 
 .. highlight:: doscon
 
-This document will guide you through installing Python 3.7 and Django on
+This document will guide you through installing Python 3.8 and Django on
 Windows. It also provides instructions for setting up a virtual environment,
 which makes it easier to work on Python projects. This is meant as a beginner's
 guide for users working on Django projects and does not reflect how Django
 should be installed when developing patches for Django itself.
 
-The steps in this guide have been tested with Windows 7, 8, and 10. In other
+The steps in this guide have been tested with Windows 10. In other
 versions, the steps would be similar. You will need to be familiar with using
 the Windows command prompt.
 
 Install Python
 ==============
 
 Django is a Python web framework, thus requiring Python to be installed on your
-machine. At the time of writing, Python 3.7 is the latest version.
+machine. At the time of writing, Python 3.8 is the latest version.
 
 To install Python on your machine go to https://python.org/downloads/. The
 website should offer you a download button for the latest Python version.
-Download the executable installer and run it. Check the boxes next to ``Install
-launcher for all users (recommended)`` and ``Add Python 3.7 to PATH`` then
```

Comment:
The advice "Add Python 3.7 to PATH" also exists in `docs/intro/contributing.txt`, as the advice is being removed from this article, should this same change also be applied there?

https://github.com/django/django/blob/8dabdd2cc559a66b519e2a88b64575d304b96ebe/docs/intro/contributing.txt#L78-L81

Reply:
I've changed this admonition to link back to the more detailed Windows/Python install docs.